.github/workflows/release-please.yml

```diff
@@ -46,6 +46,29 @@ jobs:
                   npm run build
               if: ${{ steps.release.outputs.releases_created == 'true' }}
 
+            #-----------------------------------------------------------------------------
+            # NOTE: This script currently doesn't do anything. It just outputs the
+            # release information to the console. We will do this for a few releases
+            # to make sure everything is working correctly before we switch to use this
+            # script exclusively.
+            #-----------------------------------------------------------------------------
+
+            - name: Publish using new script
+              run: node scripts/publish.js --dry-run
```

Comment:
you should put this in `package.json` and use `node --run publish` it's will be easier to maintain